### ISSUE TITLE
Missing `import org.gradle.util.VersionNumber` in the `gradle.groovy` file

### ISSUE THREAD

**Issue by mahmuodmasri** [unimodules/react-native-unimodules#50]
When running `react-native run-android` I'm getting the following error, 

```
FAILURE: Build failed with an exception.

* Where:
Script '/Users/mahmoudmasri/Projects/hypatia/iwe-client/node_modules/react-native-unimodules/gradle.groovy' line: 154

> Could not get unknown property 'VersionNumber' for settings 'android' of type org.gradle.initialization.DefaultSettings.
```

which can be fixed easily by adding `import org.gradle.util.VersionNumber` to the `react-native-unimodules/gradle.groovy` file

**Comment by djMax** [comment]
I'm not sure why this doesn't affect everyone, that groovy file certainly uses VersionNumber and certainly doesn't import it.

**Comment by MarcoPolo** [comment]
It depends on what version of gradle you're using. I think if you're on a version past 5.1.1 VersionNumber is not in the global scope. [Related](https://github.com/EddyVerbruggen/nativescript-plugin-firebase/pull/1250)

I'm on 5.4.1 and I see this error.

**Comment by maddijoyce** [comment]
@MarcoPolo if it gets added to the `gradle.groovy` file, will it break on older versions of gradle?

**Comment by MarcoPolo** [comment]
I don't think so. https://github.com/gradle/gradle/blob/v3.2.0/subprojects/core/src/main/java/org/gradle/util/VersionNumber.java shows that class has been there for a while (definitely 3.2 which is as far back as the git tags go in that repo)

**Comment by tsapeta** [comment]
Pull request fixing this issue (#57) has been merged and published in version `0.5.4`. Sorry it took a bit of time 😅
